### PR TITLE
Tracking bugs that were filed with tracking flags from start, #1212

### PR DESCRIPTION
**Pull request by kushagra414**
This commit is corresponding to issue #1212.
Please Review

### CODE REVIEW

File: bugbug/models/tracking.py [hunk 1]
```diff
@@ -100,13 +100,27 @@ def get_labels(self):
...
             bug_id = int(bug_data["id"])
 
-            for entry in bug_data["history"]:
-                for change in entry["changes"]:
-                    if change["field_name"].startswith("cf_tracking_firefox"):
-                        if change["added"] in ["blocking", "+"]:
-                            classes[bug_id] = 1
-                        elif change["added"] == "-":
-                            classes[bug_id] = 0
+            flag = False
```
Comment (marco-c) [inline bugbug/models/tracking.py]: Nit: rename this variable to something more meaningful, e.g. flag_found

File: bugbug/models/tracking.py [hunk 2]
```diff
@@ -100,13 +100,27 @@ def get_labels(self):
...
+            keys = bug_data.keys()
```
Comment (marco-c) [inline bugbug/models/tracking.py]: No need to store the iterator in a variable, just use it directly

File: bugbug/models/tracking.py [hunk 3]
```diff
@@ -100,13 +100,27 @@ def get_labels(self):
...
+            tracking_list = [
```
Comment (marco-c) [inline bugbug/models/tracking.py]:
```suggestion
            tracking_flags = [
```

File: bugbug/models/tracking.py [hunk 4]
```diff
@@ -100,13 +100,27 @@ def get_labels(self):
...
+                ele for ele in keys if ele.startswith("cf_tracking_firefox")
```
Comment (marco-c) [inline bugbug/models/tracking.py]:
```suggestion
                flag for flag in flags if flag.startswith("cf_tracking_firefox")
```

File: bugbug/models/tracking.py (hunk 3 above)
Comment (kushagra414) [reply bugbug/models/tracking.py]: Shall I create a new PR or commit chages in this PR directly?

**Comment by kushagra414** [comment]
Shall I create a new PR or commit chages in this PR directly?

**Comment by marco-c** [comment]
> Shall I create a new PR or commit chages in this PR directly?

Definitely update this PR directly, it would be chaotic otherwise :D

**Comment by kushagra414** [comment]
bugbug lint task failed with
```
black....................................................................Failed
hookid: black
```
Why?

**Comment by marco-c** [comment]
You have linting failures. Did you follow https://github.com/mozilla/bugbug#auto-formatting?

**Comment by kushagra414** [comment]
Is this OK?

File: .gitignore [hunk 5]
```diff
@@ -18,6 +18,7 @@ http_service/models/*model*
 data/
 sheets/
 
+.mypy_cache/
```
Comment (marco-c) [inline .gitignore]: Please remove this change, it's unrelated to the PR.

File: bugbug/models/tracking.py [hunk 6]
```diff
@@ -100,13 +100,28 @@ def get_labels(self):
...
-                            classes[bug_id] = 1
-                        elif change["added"] == "-":
-                            classes[bug_id] = 0
+            flag_found = False
+            tracking_flags = [
+                flag
+                for flag in bug_data.keys()
+                if flag.startswith("cf_tracking_firefox")
+            ]
+            for tracker in tracking_flags:
```
Comment (marco-c) [inline bugbug/models/tracking.py]:
```suggestion
            for tracking_flag in tracking_flags:
```

**Comment by kushagra414** [comment]
I Made those changes you asked for.
Shall I create a new PR for adding mypy_cache/ to gitignore file?

**Comment by marco-c** [comment]
>  Shall I create a new PR for adding mypy_cache/ to gitignore file?

Yes, thanks!

**Comment by kushagra414** [comment]
Thank You for your guidance.
I learned a lot!

**Pull request merged**